openstudiocore/src/model_editor/Qt.i

```diff
@@ -161,12 +161,12 @@ class QComboBox : public QWidget
 
 //class QString{};
 
-//%extend QString{
-//  // to std::string
-//  std::string __str__() const{
-//    return toString(*self);
-//  }
-//}
+%extend QString{
```

Comment:
@kbenne this just reminded me that ModelEditor.so (used by SU Plugin) will still need to link to static Qt (unless we want to install Qt binaries which is a pain because SU is the host application).  So the app repo can still build against both static and shared Qt, or we have to mess with the SU installer.  Probably start by building both and then fix installer later